Source/Arrow.swift

```diff
@@ -7,8 +7,6 @@
 //
 
 import Foundation
-import CoreGraphics
```

Comment:
@adrianbrink @s4cha CoreGraphics import is required here to use `CGFloat` type. It will fail to compile without that if Arrow is added to Playground for example.